### ISSUE TITLE
Champ "accessoires" dans les outils

### ISSUE THREAD

**Issue by MichelDwo** [lairdubois/lairdubois#83]
Salut,

J'aimerais suggérer l'ajout un champ "accessoires" pour les outils (je n'ai pas trouvé de meilleur nom pour ce que j'aurais appelé "linked-to-this-tool").

Certains outils comme les scies à onglet ou les scies sur table peuvent être montées sur piètement fixe ou mobile.
Des visseuses peuvent recevoir différents mandrins pas forcément vendus avec par défaut.
Certaines marques proposent de monter leurs outils sous table (sur des modules plus forcément produits ou ayant été remplacés)

Ce n'est donc pas pour décrire les consommables mais pour tout ce qui permet l'exploitation de l'outil autrement, plus facilement, ...

Peut être ajouté sous 3utilisation" dans "Divers" ?

(Je ne suis plus assez au point niveau dev pour proposer une pull request, d'où cette issue).

**Comment by bbeaulant** [comment]
Salut Michel,

Si je comprends bien tu propose que ces accessoires soient pleinement intégrés à la fiche principale de l'outil.
Ca ne risque pas d'être un peu "lourd" à gérer de pouvoir y mettre toutes les informations voulues ? Ou trop simpliste de réduire à du simple texte ?

Peut-être faudrait-il que les les accessoires soient simplement des nouvelles fiche qui ont une forme de lien avec la fiche de leur outil correspondant ?

**Comment by MichelDwo** [comment]
`Peut-être faudrait-il que les les accessoires soient simplement des nouvelles fiche qui ont une forme de lien avec la fiche de leur outil correspondant ?`

Merci d'avoir su interpréter ce que j'ai voulu dire: c'est tout à fait ça l'idée :)

**Comment by bbeaulant** [comment]
Oui, à l'image de ce qui a été fait pour les plugins dans la rubrique Logiciel.

Mais, ça sera pas si simple pour les outils car un simple nom ne suffit pas à les identifier. C'est pour ça que c'est resté en attente de trouver le temps d'y réfléchir pour le moment :)

**Comment by MichelDwo** [comment]
Je garde ça dans un coin de tête pour y réfléchir également dans ce cas :)

Merci beaucoup pour tes réponses !
